client/src/components/AppointmentCreateForm.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+
+import "react-datetime/css/react-datetime.css";
+
+import React, { useState } from "react";
+import { gql, useMutation } from '@apollo/client';
+
+import Datetime from "react-datetime";
+import { FETCH_APPOINTMENTS } from "./PickupList";
+import { useHistory } from "react-router-dom";
+
+const CREATE_APPOINTMENT = gql`
+  mutation createAppointment($startTime:Int! $endTime:Int! $address:String! $note:String! ){
+    createAppointment(startTime:$startTime, endTime:$endTime, address:$address, note:$note){
+      id,
```

Comment:
this part (from line 15 to 22) specifies what's returned about the appointment that was just created, and you don't need them if you are not reading from the response to render the page. Try removing them and see what happens.

Reply:
It works if I delete, just need to leave one entity name there otherwise the whole mutation does not work properly. It showed me a warning.